### ISSUE TITLE
How do I create a tree control in the markup language?

### ISSUE THREAD

**Issue by Thraka** [rds1983/Myra#360]
I can't seem to figure out. Looking at the all controls example, it seems its generated in code. Any plans to add it to the markup?

**Comment by rds1983** [comment]
Yeah, the tree isnt supported in the MML yet. Eventually it'll be supported.

**Comment by Thraka** [comment]
Want me to leave this open to track?

**Comment by rds1983** [comment]
Yes, leave it open, thanks.
